spryker/usr/local/share/container/baseimage-45.sh

```diff
@@ -4,7 +4,8 @@ source /usr/local/share/spryker/spryker_functions.sh
 alias_function do_build_permissions do_spryker_build_permissions_inner
 do_build_permissions() {
   do_spryker_build_permissions_inner
-  do_spryker_build
+  do_spryker_directory_create
+  do_spryker_config_create
```

Comment:
if do_spryker_build is running these anyway, do we need to do these here? Also what happens if in dev mode if the other parts of the do_spryker_build aren't run on the mount? Do you think it can be just that the call to do_spryker_build is moved to do_development_start?

Reply:
> Do you think it can be just that the call to do_spryker_build is moved to do_development_start?

I guess that could be the solution. @kierenevans @richardmiller do you think it would be a problem for the local development?